### DISCUSSION TITLE
ytdl error

### DISCUSSION THREAD

**Discussion by cancundeyingzi** [hooke007/mpv_PlayKit#368] [Q&A 问答]
![image](https://github.com/hooke007/MPV_lazy/assets/73635883/fffe20f9-9a0e-4c90-af59-ca7d3cc23431)
youtube视频放不出,已经尝试各种代理配置,tun网卡模式

**Comment by cancundeyingzi** [comment]
[mpv-lazy.log](https://github.com/hooke007/MPV_lazy/files/12642487/mpv-lazy.log)

**Reply by hooke007** [reply]
图上一眼看过去全是ytdlhook脚本相关的问题，和mpv没啥关系

**Reply by cancundeyingzi** [reply]
不过看上去可能和配置文件有关?

**Reply by cancundeyingzi** [reply]
似乎获取不到饼干()

**Reply by hooke007** [reply]
读不到cookie又不影响能不能播放

去掉测测，报错尽量no config开始，然后往上加变量，而不是反过来

**Reply by cancundeyingzi** [reply]
试过了，配置文件好像有问题，现在clash开tun网卡模式正常，系统代理还是不行，加了那个什么端口的两行配置也不行

**Reply by 921j** [reply]
大佬，请教一下如何获得 mpv 当前播放的媒体文件的路径，不要文件名？
只知道一个 ${path} 参数，但这个是包含文件名及扩展名的完整路径，我只想要路径不要文件名及扩展名该怎么写呢？求赐教！
